src/vs/workbench/browser/parts/notifications/notificationsStatus.ts

```diff
@@ -79,6 +83,19 @@ export class NotificationsStatus extends Disposable {
 			showBeak: this.isNotificationsCenterVisible
 		};
 
+		const isDoNotDisturbEnabled = this.notificationService.getDoNotDisturbMode();
```

Comment:
No need for a local variable, can inline the call below where it is used. You also do not need to triple equals for `true`, just do `if (this.notificationService.getDoNotDisturbMode())`